Azure/Start-AzureStorageEmulator/Start-AzureStorageEmulator.psm1

```diff
@@ -13,17 +13,17 @@
 function Start-AzureStorageEmulator
```

Comment:
Is this script needed? I thought ASE has been deprecated for some time.

Reply:
We're not using it anymore, I'll remove it.